### DISCUSSION TITLE
SpiralLinux doesn't show other Spiral installations in Grub-Menu

### DISCUSSION THREAD

**Discussion by newbie992** [SpiralLinux/SpiralLinux-project#195] [Q&A]
I installed 3 SpiralLinux desktops for testing on my PC.
KDE, Xfce and LXQt.
The last installed Spiral creates the grub menu ( my PC is Uefi ) .... and the other Spiral disappears from the grub menu.

In order to boot all 2 or 3 Spiral installations, I have to use efibootmgr for putting one of my other distros (Manjaro, Endeavour, PeppermintOS) on top. I have to boot from such a distro, make "sudo update-grub" there. Then all SpiralLinux distros appear
in the grub menu and I can choose which one I want to  boot.

Is there a way to make Spiral to show other Spiral installations in its grub menu ?

**Comment by geckolinux** [comment]
Hi there, that's a limitation of Debian. There is a workaround here: https://github.com/SpiralLinux/SpiralLinux-project/issues/3

**Comment by newbie992** [comment]
Thank you geckolinux.
I did as suggested.
I am not able to make it work. 
It is too complicated for me.
The instructions like "#Mount the other OS that is installed on Btrfs and navigate to its root
cd /mnt/exampleotherbtrfsvolume" seem to be easy, but not for me.
I don't want to use your time to give more instructions.

 I will circumvent this debian bug like described. Will boot SpiralLinux from other distros because they show Spiral distros in the grub-menu without problems.

**Reply by pertsavk** [reply]
Not an actual answer to your question, but rather a suggestion for another workaround. Try installing rEFInd. https://www.rodsbooks.com/refind/

**Reply by newbie992** [reply]
@pertsavk,
thank you for suggesting rEFInd.
I installed it, but since I have 6 to 8 linux-distros installed , always besides Win10, the boot-menu of rEFInd is a mess.
Some distros appear more than once ( instead of 1 start button for a distro, rEFInd is putting 2 buttons there ) .
I deleted rEFInd because of this.
